extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -105,6 +103,20 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async showPreviewFile(books: BookTreeItem[], bookPath: string): Promise<void> {
+		let book = books.filter(book => book.root.indexOf(bookPath.replace(/\\/g, '/')) > -1)[0];
+		const readmeMarkdown: string = path.join(bookPath, 'content', book.tableOfContents[0].url.concat('.md'));
+		const readmeNotebook: string = path.join(bookPath, 'content', book.tableOfContents[0].url.concat('.ipynb'));
+		const markdownExists = fs.existsSync(readmeMarkdown);
+		const notebookExists = fs.existsSync(readmeNotebook);
```

Comment:
Why are you reverting this? In general we should avoid using the sync methods whenever we can

Reply:
for some reason when am debugging the promsify throws an error. So for testing I changed it and forgot to revert it back.

Reply:
changed it back